### ISSUE TITLE
mirror and delay

### ISSUE THREAD

**Issue by HB9SIU** [w4mmp/MSCC-PW#3]
mirror-and-delay

already is ok !?? But....

Sorry, i have never used GetHub befor today. It is my first contribution.

First and important: Thanks a lot for your mainpower and help, Ron.
****************************************************************************
I've had the hardware Proficio in operation for weeks. At the beginning with the software "MSCC" under the files "Proficio". Starting problem with driver. Fast and successful help through "MultusSDR". With the appearance of the variant "MSCC-PW" this app is installed.

Working with the "MSCC" app is basically possible,... but 2 phenomena show up here with me.

There is a signal e.g. 3766 kc (CW) from a Hf-Generator at the HF input at "Proficio" Level -60dBm.
appears in the spectrum. e.g. not only the line at 3766 kc, but also another line at about 3752 kc.
If the receiving frequency is changed by the Proficio, the frequency lines in the spectrum change as a mirror image.

This phenomenon also occurs with real received signals. Maybe correctable with RX I/Q ?? But don't know how. 
Description is still missing in the manual.

Initial situation: reception 80m band, mode: LSB Commands for transmission and frequency change in the band
are carried out immediately (less than 0.5 seconds). Then... Frequency change to 20m band. Mode: CW: 5 seconds transmission. All subsequent commands such as frequency change (band) as well as changes in small steps or send commands are executed with a delay of up to 4 seconds.

If the "MSCC.PW" app is closed, the main window goes out immediately, but not the two sub-windows of the spectrum and waterfall. The voice information is still audible, even if the main window has already been deleted, but the two sub-windows "Spectrum" and "Waterfall" are still active. Only after about 10 -15 seconds does the "MSCC" report with error messages. Summary: All operations are executed successfully, but always with a delay of 4-5 seconds. Extremely difficult operation !!

Thanks for any help. 73 de HB9SIU, John

[System-INFO.txt](https://github.com/w4mmp/MSCC-PW/files/9387329/System-INFO.txt)

![MSCC_SYS](https://user-images.githubusercontent.com/45912766/185752303-43cda02b-c65c-4864-b6c3-2550c55c0c47.JPG)
[mscc.log](https://github.com/w4mmp/MSCC-PW/files/9387330/mscc.log)
[MSCC-Spectrum.exe.log](https://github.com/w4mmp/MSCC-PW/files/9387332/MSCC-Spectrum.exe.log)
[MSCC-Waterfall.exe.log](https://github.com/w4mmp/MSCC-PW/files/9387333/MSCC-Waterfall.exe.log)
[mscc-pw.exe.log](https://github.com/w4mmp/MSCC-PW/files/9387499/mscc-pw.exe.log)

**Comment by w4mmp** [comment]
Hello John,
Thanks for the issue report.  I will follow up soon.
73,
Ron

**Comment by w4mmp** [comment]
What operating system?   Windows 10?

**Comment by w4mmp** [comment]
> There is a signal e.g. 3766 kc (CW) from a Hf-Generator at the HF input at "Proficio" Level -60dBm.
appears in the spectrum. e.g. not only the line at 3766 kc, but also another line at about 3752 kc.
If the receiving frequency is changed by the Proficio, the frequency lines in the spectrum change as a mirror image.

What is the signal level of the image?  Does this happen on other bands.  Can you run a test signal on perhaps 40M?

At any rate,  if it is an image, it should at +24KHz not 14KHz.

73,
Ron

**Comment by w4mmp** [comment]
Hello John,
I was able to reproduce the slow down issue by quickly moving through the operating modes (clicking the mode button).  Now to determine why this is happening.

**Comment by w4mmp** [comment]
Hi John,
I'm going to close this issue.  If you continue to have the slow down issue, please create a new issue.
Ron,
W4MMP

**Comment by HB9SIU** [comment]
Yes WIN10  More Details read in File  "System-INFO.txt" Am 20.08.2022 21:45 schrieb W4MMP ***@***.***>:
What operating system?   Windows 10?
